docs/examples/bayesian_optimisation.py

```diff
@@ -216,10 +217,10 @@ def return_optimised_posterior(
 
     opt_posterior, history = gpx.fit(
         model=posterior,
-        objective=negative_mll,
-        train_data=data,
-        optim=ox.adam(learning_rate=0.01),
-        num_iters=1000,
+        train_data=D,
```

Comment:
@henrymoss  Maybe, this changed from `train_data=data` to `train_data=D`?

Reply:
its decision_making that is rubbish